frontend/src/queries/schema/schema-assistant-messages.ts

```diff
@@ -124,6 +126,7 @@ export interface NotebookUpdateMessage extends BaseAssistantMessage {
     notebook_id: string
     content: ProsemirrorJSONContent
     tool_calls?: AssistantToolCall[]
+    stage_notebooks?: DeepResearchNotebookInfo[]
```

Comment:
Does it make sense to have both `notebook_id` and `stage_notebooks`? It seems redundant, and also it probably doesn't make sense to call it `DeepResearchNotebookInfo` as notebooks are also being used by session recordings summary. We should extend this in a way that is also consistent with that feature.